source/help/apps/desktop-changelog.rst

```diff
@@ -1,7 +1,7 @@
 Desktop Application Changelog
 ========================================
 
-Release v4.4.0
+Release v4.4.1
```

Comment:
@amyblais Another option is being consistent with server releases, e.g. "v4.3", "v4.2": https://docs.mattermost.com/administration/changelog.html